dipy/reconst/dki.py

```diff
@@ -0,0 +1,1326 @@
+#!/usr/bin/python
+""" Classes and functions for fitting tensors """
+from __future__ import division, print_function, absolute_import
+
+import warnings
+
+import numpy as np
+
+import scipy.optimize as opt
+
+from dipy.reconst.dti import (fractional_anisotropy, geoderic_anisotropy,
```

Comment:
Typo: `geoderic_anisotropy`-->`geodesic_anisotropy`, this is causing the Travis build failure